language/bytecode_verifier/bytecode_verifier_tests/src/unit_tests/mod.rs

```diff
@@ -7,3 +7,59 @@ pub mod duplication_tests;
 pub mod resources_tests;
 pub mod signature_tests;
 pub mod struct_defs_tests;
+
+use vm::file_format::*;
+
+/// Creates a module with an empty function definition and an empty struct definition. Convenient
```

Comment:
`file_format::dummy_procedure_module` does something fairly similar to this. Might make sense to extend that testing code to allow for customizing the procedure signature + fields?

Reply:
I'll take a look.